ivy/functional/ivy/sorting.py

```diff
@@ -63,7 +63,7 @@ def argsort(
     :code:`ivy.Array` or :code:`ivy.NativeArray` instances, as shown in the type hints
     and also the examples below.
 
-    Functional Examples
+    Examples
     -------------------
```

Comment:
please update the number of `-` to match the number of characters in the header i.e. `Examples`